cloudformation/dns/template.yaml

```
@@ -46,6 +46,13 @@ Resources:
     Properties:
       Name: data-exchange-catalogue-detailed.account.gov.uk
 
+  DataRadarPublicHostedZone:
+    Type: AWS::Route53::HostedZone
+    DeletionPolicy: Retain
+    UpdateReplacePolicy: Retain
+    Properties:
+      Name: data-radar.account.gov.uk
```

I think this should have an .internal in too 
```suggestion
      Name: data-radar.internal.account.gov.uk
```